src/morello-install/README.md

```diff
@@ -121,11 +121,12 @@ You can press Enter to proceed, or wait for the countdown timer to finish:
    Autoboot in 10 seconds. [Space] to pause
 ```
 
-Note that the installer will only run on the configured primary console, which
-defaults to serial. If you are using a video console instead, press 5 or C
-until it displays "5. Cons: Video". This is only required during installation.
+Note the seperate instances of the installer will run on both the video and
+serial consoles.  Care should be taken to ensure that only one is uses as
+no mechanism is implemented to prevent one from destroying the other's work.
```

Comment:
```suggestion
Note that seperate instances of the installer will run on both the video and
serial consoles.  Care should be taken to ensure that only one is used as
no mechanism is implemented to prevent one from destroying the other's work.
```